modules/core/src/main/java/org/apache/ignite/IgniteSystemProperties.java

```diff
@@ -1342,6 +1343,15 @@
      */
     public static final String IGNITE_DUMP_TX_COLLISIONS_INTERVAL = "IGNITE_DUMP_TX_COLLISIONS_INTERVAL";
 
+    /**
+     * Newline separated list of keys that {@link DistributedMetaStorage} will skip for any operations:
```

Comment:
Why do we want to use a newline to separate list items?
It may be inconvenient to specify a multiline value in the bash script.
Let's use a comma as a separator like it done in `IGNITE_TCP_DISCOVERY_ADDRESSES`

Reply:
Fixed with comma